### PR TITLE
EUI-5498

### PR DESCRIPTION
**Pull request by maliumer**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5498

### Change description ###
Welsh language toggle

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/phase-banner/phase-banner.component.html [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <div class="hmcts-header__container">
   <div class="govuk-phase-banner">
-      <p class="govuk-phase-banner__content"><strong class="govuk-tag govuk-phase-banner__content__tag ">
+    <a href="#" href="javascript:void(0)" (click)="toggleLanguage('cy')" *ngIf="currentLang == 'en'" class="language">Gymraeg</a>
```
Comment (rayliangatos) [inline src/app/components/phase-banner/phase-banner.component.html:3]: duplicated href, the one href="#" can be deleted?

File: src/app/components/phase-banner/phase-banner.component.html [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+    <a href="#" href="javascript:void(0)" (click)="toggleLanguage('en')" *ngIf="currentLang == 'cy'" class="language">English</a>
```
Comment (rayliangatos) [inline src/app/components/phase-banner/phase-banner.component.html:4]: duplicated href, the one href="#" can be deleted?

File: src/app/components/phase-banner/phase-banner.component.html [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+    <a href="javascript:void(0)" (click)="toggleLanguage(currentLang == 'cy' ? 'en' : currentLang == 'en' ? 'cy' : 'en')" class="language">{{currentLang == 'cy' ? 'English' : currentLang == 'en' ? 'Cymraeg' : 'English' }}</a>
+    <p class="govuk-phase-banner__content"><strong class="govuk-tag govuk-phase-banner__content__tag ">
               {{type}}
           </strong>
           <span class="govuk-phase-banner__text">
-              This is a new service – your
+              {{'This is a new service – your' | rpxTranslate }}
               <a class="govuk-link" href="https://www.smartsurvey.co.uk/s/CCDSurvey/" target="_blank" rel="noopener noreferrer">
-                feedback <span class="visuallyhidden">(Opens in a new window)</span>
+                {{'feedback' | rpxTranslate }}<span class="visuallyhidden">(Opens in a new window)</span>
```
Comment (MateiHMCTS) [inline src/app/components/phase-banner/phase-banner.component.html:10]: I think "(Opens in a new window)" needs to be translated as well

File: src/app/shared/shared.module.ts [hunk 4]
```diff
@@ -11,6 +11,7 @@ import * as fromAppDirectives from '../directives';
 import { TaskSupervisorGuard } from '../guards/task-supervisor.guard';
 import { HealthCheckGuard } from './guards/health-check.guard';
 import { HealthCheckService } from './services/health-check.service';
+import { InfoMessageCommService } from './services/info-message-comms.service';
```
Comment (rayliangatos) [inline src/app/shared/shared.module.ts:14]: is this redudant?

File: src/app/components/phase-banner/phase-banner.component.ts [hunk 5]
```diff
@@ -7,9 +8,17 @@ import { Component, Input, OnInit } from '@angular/core';
...
+    return this.langService.language;
+  }
+
+  constructor(private readonly langService: RpxTranslationService) { }
+
+  public ngOnInit(): void {
+  }
 
-  ngOnInit() {
+  public toggleLanguage(lang: RpxLanguage) {
```
Comment (rayliangatos) [inline src/app/components/phase-banner/phase-banner.component.ts:20]: could this return with void?

**Review by MateiHMCTS — CHANGES_REQUESTED** [review]

**Review by rayliangatos — APPROVED** [review]

**Review by MateiHMCTS — APPROVED** [review]

**Pull request merged**